README.md

```diff
@@ -22,6 +22,7 @@ This component is currently invoked manually after an HCA submission.
 ```
 docker run -v $PWD:/output \
 --env INGEST_API_URL=http://api.ingest.archive.data.humancellatlas.org/ \
+--env INGEST_API_GCP={ "type": "service_account", "project_id": "...", "private_key_id": "...", "private_key": "...", "client_email": "...", "client_id": "...", "auth_uri": "https://accounts.google.com/o/oauth2/auth", "token_uri": "https://oauth2.googleapis.com/token", "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs", "client_x509_cert_url": "..." } \
```

Comment:
could also be sourced from a file, but the user should be cautious not to check in the gcp file